build.sbt

```diff
@@ -1,18 +1,25 @@
+import ohnosequences.sbt.SbtS3Resolver._
+import com.amazonaws.services.s3.model.CannedAccessControlList
+
 name := "prequel"
 
-version := "0.3.9"
+version := "0.3.11"
 
-organization := "net.noerd"
+organization := "com.celtra"
 
 scalaVersion := "2.10.3"
 
+scalacOptions ++= Seq(
+    "-Xmax-classfile-name", "140"
```

Comment:
No, this is not necessary. It's required to compile on ecryptfs, where I compiled it. It has no impact on anything else, so it stayed there